packages/adapter/lib/adapter/adapter.js

```diff
@@ -24,13 +24,15 @@ const fs = require('fs-extra');
 const extend = require('node.extend');
 const util = require('util');
 const os = require('os');
+const path = require('path');
 const EventEmitter = require('events').EventEmitter;
 const {tools} = require('@iobroker/js-controller-common');
 const pidUsage = require('pidusage');
 const deepClone = require('deep-clone');
 const {EXIT_CODES} = require('@iobroker/js-controller-common');
 const {PluginHandler} = require('@iobroker/plugin-base');
-const controllerVersion = require(tools.getControllerDir() + '/package.json').version;
+// local version is always same as controller version, since lerna exact: true is used
+const controllerVersion = require(path.join(__dirname, '..', '..', 'package.json')).version;
```

Comment:
Might also be possible with `require("@iobroker/js-controller-adapter/package.json")` in case we end up changing the directory structur at some point

Reply:
will this work out from the same module?

Reply:
I will try it

Reply:
It should AFAIK